### DISCUSSION TITLE
Ground Items not displaying randomly, ALT key triggers yellow boxes on screen but no longer displays items or ability to toggle them

### DISCUSSION THREAD

**Discussion by SlimReaper420** [runelite/runelite#17853] [Q&A]
My Ground Items are completely gone whether they are on the list or not, the plug-ins are on and havent been changed either. Double tapping or holding ALT no longer displays or toggles the items.

**Comment by cdfisher** [comment]
Does this issue occur for you in [safe mode](https://github.com/runelite/runelite/wiki/RuneLite-Launcher-Configuration#safe-mode)?
